docs/hugo/content/introduction/upgrading.md

```diff
@@ -52,3 +52,27 @@ helm upgrade --devel --version v2.0.0-beta.3 aso2 aso2/azure-service-operator \
 ## Supported Versions
 
 We provide best effort support via GitHub for ASO, focusing on the latest version and the previous version. If you're running an older version, we may ask you to upgrade to the latest release before we can help you.
+
+## Downgrading
+
+If you encounter an issue after an upgrade of ASO that can't be addressed by any other means, you may want to downgrade to the prior version. This isn't an officially supported scenario, but we'll try to provide some guidance here.
+
+Summary: If the new version of ASO didn't change the stored version of your resource, then you should be able to downgrade without issue. If the new version of ASO did change the stored version of your resource, then you will need to manually fix things up.
+
+Each ASO resource is stored within the cluster using a canonical storage (or hub) version, regardless of the actual version used for interaction with the cluster or with Azure. These stored versions are based on the latest Stable Azure API version of that resource. (See [Resource Versioning](https://azure.github.io/azure-service-operator/design/versioning/) for all the background on this design decision.)
+
+When a new version of ASO adds support for a later version of an already supported resource (say, adding support for the latest release of Azure Redis, v2022-06-01, in addition to the existing support for v2021-12-01), then the new version of ASO will use the newer API version of the resource as the storage version. In this situation, downgrading ASO to the prior version will orphan the resource, as the older version of ASO is unaware of the newer storage version and is unable to interact with it. This means that any resources created with the newer version of ASO will be unable to be managed by the older version of ASO. They will also be inaccessible via `kubectl` or any other tool.
```

Comment:
minor: Should refer to CRD versions here, which would be something like `v1api20220601`, rather than ARM API versions, since you're talking about CRDs

Reply:
Done.